winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,65 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', '$state', '$stateParams', 'claimService', 'formTemplateService',
+  function($scope, $state, $stateParams, claimService, formTemplateService) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+
+    $scope.allForms = formTemplateService; // returns an object {formId -> {...}}
```

Comment:
just a note. For some reason, `formTemplateService` is not found by angular when you just run
`gulp` (not `gulp build`).
Running gulp creates a local webserver to host your webapp and will auto-update whenever you change a css property, template, js, etc so you can view your changes without having to reload pages.
`gulp build` just does all the proprocessing stuff and moves those files to the `build` folder.